### PR TITLE
[MPA Detection] enable saliency map detection

### PR DESCRIPTION
**Pull request by eugene123tw**
Correponded ticket for MPA: https://github.com/openvinotoolkit/model_preparation_algorithm/pull/24

### CODE REVIEW

**Comment by eugene123tw** [comment]
run ote-test

**Comment by eugene123tw** [comment]
run ote-test

**Comment by eugene123tw** [comment]
https://ci-ote.iotg.sclab.intel.com/job/ote/job/pr-ote-test/434/

**Comment by eugene123tw** [comment]
run ote-test

**Comment by eugene123tw** [comment]
https://ci-ote.iotg.sclab.intel.com/job/ote/job/pr-ote-test/435/

File: external/model-preparation-algorithm/mpa_tasks/apis/detection/task.py [hunk 1]
```diff
@@ -82,19 +83,48 @@ def infer(self,
...
+    def _infer_detector(self, dataset: DatasetEntity, 
+                        inference_parameters: Optional[InferenceParameters] = None) -> Tuple[Iterable, float]:
+        """ Inference wrapper
+
+        This method triggers the inference and returns `prediction_results` zipped with prediction results, 
+        feature vectors, and saliency maps. `metric` is returned as a float value if InferenceParameters.is_evaluation 
+        is set to true, otherwise, `None` is returned.
+
+        Args:
+            dataset (DatasetEntity): the valuation or test dataset to be inferred with
```
Comment (JihwanEom) [inline external/model-preparation-algorithm/mpa_tasks/apis/detection/task.py]: typo for valuation -> validation?

**Review by JihwanEom — DISMISSED** [review]
LGTM.

**Review by harimkang — CHANGES_REQUESTED** [review]
The code contents seem to be clear. But the change in the submodule seems a little strange, is it right for the develop branch? It seems necessary to match the submodule commit.

**Review by harimkang — DISMISSED** [review]
I was mistaken about the comment above. I mistook both the ote and mpa for develop branch.
LGTM

**Comment by harimkang** [comment]
Pretty big PR just got merged. Please correct the MPA submodule (develop branch) and MPA task as part of it has been changed.

**Comment by eugene123tw** [comment]
run ote-test

**Comment by eugene123tw** [comment]
https://ci-ote.iotg.sclab.intel.com/job/ote/job/pr-ote-test/436/

**Comment by eugene123tw** [comment]
@harimkang Thanks for the review. I have merged the latest commit from develop branch

**Review by harimkang — APPROVED** [review]
LGTM

**Review by chuneuny-emily — APPROVED** [review]

**Review by goodsong81 — APPROVED** [review]

**Pull request merged**